lighthouse-core/audits/long-tasks.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit.js');
+const NetworkRecords = require('../computed/network-records.js');
+const i18n = require('../lib/i18n/i18n.js');
+const MainThreadTasks = require('../computed/main-thread-tasks.js');
+const BootupTime = require('./bootup-time.js');
+
+const UIStrings = {
+  /** Title of a diagnostic LH audit that provides details on the longest running tasks that occur when the page loads. */
+  title: 'Long main-thread tasks',
```

Comment:
@connorjclark I have some bad news 😬 

I'd have to disagree with the conclusion in https://github.com/GoogleChrome/lighthouse/pull/10736#discussion_r423389377 based on informativeness. We do the exact same thing of "just showing" the critical request chains but everything that is listed is *bad* and shouldn't be there so we still use a prescriptive title of "Avoid chaining critical requests", even if it's not scored. I liked `Avoids long main-thread tasks` :)